packages/nocodb/src/lib/noco/meta/NcMetaMgr.ts

```diff
@@ -1480,7 +1481,9 @@ export default class NcMetaMgr {
         case 'xcExportAsCsv':
           result = await this.xcExportAsCsv(args, req, res);
           break;
-
+        case 'xcDownloadAttachments':
```

Comment:
We're not using NcMetaMgr.ts in the coming release. We put all new APIs under [here](https://github.com/nocodb/nocodb/tree/develop/packages/nocodb/src/lib/noco/meta/api).